omod/src/main/java/org/openmrs/module/adminui/page/controller/myaccount/ChangeDefaultsPageController.java

```diff
@@ -68,6 +79,35 @@ public UserDefaults getUserDefaults(@RequestParam(value = "defaultLocale", requi
     	return new UserDefaults(defaultLocale, proficientLocales);
     }
 
+    public static Locale normalizeLocale(String localeString) {
```

Comment:
The util method isn't available in platform but its present in emrapi module. Let me use that and generate a PR. Thanks. :+1: